fp_app/config/routes.rb

```diff
@@ -2,13 +2,18 @@
   root "homepages#index"
   get "hello", to: "hello_world#index"
 
-  devise_for :users, controllers: {
-    sessions: "users/sessions",
-    registrations: "users/registrations"
-  }
-
   devise_for :planners, path: "planners", controllers: {
     sessions: "planners/sessions",
     registrations: "planners/registrations"
   }
+
+  resources :planners, only: [ :show, :edit, :update ] do
+    resources :schedules, only: [ :index, :create, :update ], controller: "planners/schedules"
```

Comment:
```suggestion
    resources :schedules, only: [ :index, :create, :update ]
```
controller の指定なくても動きそうかなと思いました（がないと動かないならそのままで）

Reply:
ありがとうございます！
schedulesコントローラがplannersディレクトリの中にあるので指定しないと、controllers/schedules_controller.rbを探すようになってしまいcontrollers/planners/schedules_controller.rbにいけなくなってしまうのでコントローラを明示的に指定しています。

Reply:
```suggestion
    resources :schedules, only: [ :index, :create, :update ], modules: :planners
```
にすると良さそうでした 🙇
参考までに